firefoxThemes/options/options.js

```diff
@@ -175,4 +193,4 @@ clickListener(druthersDarkBtn, ()=>{
 });
 
 clickListener(druthersDarkBGCheckbox, onChangeDruthersDarkBG);
-clickListener(druthersLightBGCheckbox, onChangeDruthersLightBG);
\ No newline at end of file
+clickListener(druthersLightBGCheckbox, onChangeDruthersLightBG);
```

Comment:
There's some very minor issues I found involving the secondary background button for druthers.